R/easier-pkg.R

```diff
@@ -4,8 +4,11 @@
 #'
 #'
 #' @importFrom stats cor
-#' @importFrom utils install.packages installed.packages
+#' @importFrom utils globalVariables install.packages installed.packages
```

Comment:
We should not really need `install.packages` - see other comments

Reply:
Indeed, good point! What is actually this function doing ?

Reply:
We use it in the examples to install the dependencies, but these automatically handled by the package structure.
So I guess we can just remove it